shared/lib_battery.cpp

```diff
@@ -1564,6 +1564,7 @@ void thermal_t::updateTemperature(double I, double R, double dt, size_t lifetime
 		_T_battery = implicit_euler(I, dt*HR2SEC, lifetimeIndex);
 	else
 		_message.add("Computed battery temperature below zero or greater than max allowed, consider reducing C-rate");
+	_T_battery = fmax(_T_battery, _T_room[util::yearOneIndex(_dt_hour, lifetimeIndex)]);
```

Comment:
Why maxed at room temperature or battery temp? Why can't the battery be below room temperature? Is this based on Troom being a conditioned area for battery storage?

Reply:
I don't think the battery operations can cool the battery, which was what was happening due to the large step size in the numerical methods: if the current decreased dramatically, the temperature could decrease so much (as the approximate derivative was applied across the whole hour) as to be cooling the battery. If the current falls precipitously to zero, the temperature of the battery would simply to be room temp.